src/lib/select/select.ts

```diff
@@ -297,8 +297,12 @@ export class MatSelect extends _MatSelectMixinBase implements AfterContentInit,
     },
   ];
 
+  private _hasFocus = false;
+
   /** Whether the select is focused. */
-  focused = false;
+  get focused() {
```

Comment:
Uncovered during presubmit: this makes `focused` readonly, which is a breaking change
(apparently some apps set it manually for some reason)